tom_registration/registration_flows/approval_required/views.py

```diff
@@ -38,19 +38,20 @@ def form_valid(self, form):
 
         messages.info(self.request, 'Your request to register has been submitted to the administrators.')
 
-        # TODO: Enable sending TOM Admin alert emails
-        # if settings.TOM_REGISTRATION.get('SEND_APPROVAL_EMAILS'):
-        #     try:
-        #         current_domain = Site.objects.get_current().domain
-        #         link_to_user_list = f'https://{current_domain}{reverse("user-list")}'
-        #         mail_managers(
-        #             f'Registration Request from {self.object.first_name} {self.object.last_name}',
-        #             f'{self.object.first_name} {self.object.last_name} has requested to register in your TOM. Please '
-        #             f'approve or delete this user <a href="{link_to_user_list}">here</a>.',
-        #             fail_silently=False
-        #         )
-        #     except SMTPException as smtpe:
-        #         logger.error(f'Unable to send email: {smtpe}')
+        if settings.TOM_REGISTRATION.get('SEND_APPROVAL_EMAILS'):
+            try:
+                current_domain = Site.objects.get_current().domain
+                link_to_user_list = f'https://{current_domain}{reverse("user-list")}'
+                mail_managers(
```

Comment:
There is also a [`mail_admin`](https://docs.djangoproject.com/en/5.1/topics/email/#mail-admins) function as well. 
I don't have a strong preference. The main difference is one pulls emails from the `MANAGERS` setting, while the other pulls from the `ADMINS` setting.

Reply:
Since TOMToolkit already has the concept of ADMINS, I'd suggest using that and not introducing a new concept (MANAGERS) unless someone asks for it (even if this idea of ADMINS isn't exactly the same as our current idea of what admins are (i.e. the list of Users with admin priviledge). What actually is the "ADMINS setting" here?).